buildtools/src/main/java/org/apache/pulsar/tests/ThreadDumpUtil.java

```diff
@@ -18,6 +18,7 @@
  */
 package org.apache.pulsar.tests;
 
```

Comment:
Does this line work with maven.compiler.release=8 ?

Reply:
Thanks for pointing that out. I'll change the implementation to use standard JMX API instead of com.sun.*/sun.* APIs.